### DISCUSSION TITLE
When is Airflow Helm Chart 1.16.0 expected to be released?

### DISCUSSION THREAD

**Discussion by FelipeRamos-neuro** [apache/airflow#44813] [Q&A] [answered]
I would love to be able to use the webserver HPA feature

**Comment by potiuk** [comment, marked as answer]
As discussed in https://github.com/apache/airflow/pull/43264#pullrequestreview-2398737898 - we have a number of backwards-incompatible proposals and cleanups to be done in the helm chart I **think**.

So maybe the next one is 2.0.0.

And maybe a good idea would be to relase 2.0.0 chart now - to not to add a lot of back-compatibilities in there also for Airflow 3. @jedcunningham @ephraimbuddy @utkarsharma2 @eladkal @jscheffl ->  I think we should discuss our strategy for Helm chart to slack release-management channel (and devlist for conclusions) and decide what to do,
